README.md

```diff
@@ -1 +1,184 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Google Chrome or any other browse installed in your computer. 
+
+Git installed in your computer
+
+### Setup
+
+Clone this repository to your desired folder:
+To clone this repository to your local computer, follow these dteps.
+  - Goto to location where you want to clone this repository.
+  - Right click anywhere on screen and select "git bash" here
+  - Give gollowing command in git bash
+  - git clone -b index-html-style-css-branch https://github.com/anilshah042022/hello-world.git
+
+
+### Install
+
+No installation is required to run this program except those mentioned in prerequisites.
+
+### Usage
+
+To run the project, go to the location where you have cloned gitHub repositoty
+and double click index.html file
+
+### Expected output
+You will see following output in Blue color
+
+Hello Microverse
+
+### Run tests
+
+No test runs are required
+
+### Deployment
+
+No deployment is required
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Anilkumar Shah**
+
+- GitHub: [@githubhandle](https://github.com/anilshah042022))
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **About Microverse Program**
+- [ ] **How to enroll in Microverse Program**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider giving a star to this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Student success team, My mentor, My friend Mr. Yonas Lemma and all my morning sessions members, stand-up session members and
+cohort-19 members for replying my querries
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please kindly add the LICENSE file to your root folder and perhaps customize this [template](https://choosealicense.com/licenses/mit/) as suggested earlier. The MIT license gives express permission for users to reuse code for any purpose, sometimes even if code is part of proprietary software. ℹ️ Please kindly remove the template text below also. 👍 

<img width="649" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/226878530-77f84bdb-5f80-458b-a626-285212615ca3.png">